webapp/src/components/admin_settings/jaas_section.tsx

```diff
@@ -0,0 +1,164 @@
+import {TextInput} from '../InputField';
+import {RadioField} from '../RadioField';
+
+type Props = {
+    disabled: boolean,
+    onAppIDChange: (e: React.ChangeEvent<InputElementType>) => void,
+    onApiKeyIDChange: (e: React.ChangeEvent<InputElementType>) => void,
+    onPrivateKeyChange: (e: React.ChangeEvent<InputElementType>) => void,
+    onEmbeddedChange: (e: React.ChangeEvent<InputElementType>) => void,
+    onCompatibilityChange: (e: React.ChangeEvent<InputElementType>) => void,
+    apiKey: string,
+    appID: string,
+    privateKey: string
+    embedded: boolean,
+    compatibilityMode: boolean
+};
+
+const JaaSSection = ({disabled, onApiKeyIDChange, onAppIDChange, onPrivateKeyChange, onEmbeddedChange, onCompatibilityChange, apiKey, appID, privateKey, embedded, compatibilityMode}: Props) => {
+    const EMBED_JAAS_VIDEO_INSIDE_MATTERMOST_OPTIONS = useMemo(() => [
+        {
+            value: 'true',
+            checked: embedded,
+            label: (
+                <FormattedMessage
+                    id='jitsi.true'
+                    defaultMessage={'true'}
+                />
+            )
+        },
+        {
+            value: 'false',
+            checked: !embedded,
+            label: (
+                <FormattedMessage
+                    id='jitsi.false'
+                    defaultMessage={'false'}
+                />
+            )
+        }
+    ], [embedded]);
+
+    const JAAS_COMPATIBILITY_OPTIONS = useMemo(() => [
+        {
+            value: 'true',
+            checked: compatibilityMode,
+            label: (
+                <FormattedMessage
+                    id='jitsi.true'
+                    defaultMessage={'true'}
+                />
+            )
+        },
+        {
+            value: 'false',
+            checked: !compatibilityMode,
+            label: (
+                <FormattedMessage
+                    id='jitsi.false'
+                    defaultMessage={'false'}
+                />
+            )
+        }
+    ], [compatibilityMode]);
+
+    return (
+        <div>
+            <RadioField
+                heading={
+                    <FormattedMessage
+                        id='jaas.embed-meeting'
+                        defaultMessage={'Embed JaaS video inside Mattermost:'}
+                    />
+                }
+                isInline={true}
+                options={EMBED_JAAS_VIDEO_INSIDE_MATTERMOST_OPTIONS}
+                onChange={onEmbeddedChange}
+                description={
+                    <FormattedMessage
+                        id='jaas.embed-meeting-description'
+                        defaultMessage={'(Experimental) When true, JaaS video is embedded as a floating window inside Mattermost by default.'}
+                    />
+                }
+            />
+            <RadioField
+                heading={
+                    <FormattedMessage
+                        id='jaas.enable-compatibility-mod'
```

Comment:
To match the existing translation id

```suggestion
                        id='jaas.enable-compatibility-mode'
```